### PR TITLE
Update package.json to v3.0.0

### PR DESCRIPTION
**Pull request by vakrilov**
`package.json` - dependencies and version updated.

CC @vchimev

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,76 @@
+# 3.0.0 (2017-xx-xx)
+- Compatible with Angular 4 and NativeScript 3.0
+
```
Comment (sis0k0) [inline CHANGELOG.md]: required TypeScript version 2.2 as breaking change?

File: nativescript-angular/package.json [hunk 2]
```diff
@@ -25,28 +40,38 @@
...
     "codelyzer": "~3.0.0-beta.4",
+    "nativescript-intl": "^0.0.8",
     "tns-core-modules": "internal-preview",
     "tslint": "~4.5.0",
     "typescript": "~2.2.1",
-    "zone.js": "^0.8.2"
-  },
-  "nativescript": {}
+    "rxjs": "^5.0.1",
+    "reflect-metadata": "^0.1.8",
```
Comment (vchimev) [inline nativescript-angular/package.json]: What is the correct one:
dependency - "reflect-metadata": "~0.1.8"
devDependency - "reflect-metadata": "^0.1.8"

File: nativescript-angular/package.json [hunk 3]
```diff
@@ -25,28 +40,38 @@
...
+    "@angular/animations": "~4.0.0",
     "@angular/router": "~4.0.0",
+    "@angular/forms": "~4.0.0",
+    "@angular/http": "~4.0.0",
+    "tns-core-modules": "^3.0.0 || ^3.0.0-rc.1",
     "rxjs": "^5.0.1",
-    "reflect-metadata": "~0.1.8",
-    "punycode": "1.3.2",
-    "querystring": "0.2.0",
-    "url": "0.10.3"
```
Comment (vchimev) [inline nativescript-angular/package.json]: Don't we need anymore:
```
    "punycode": "1.3.2",
    "querystring": "0.2.0",
    "url": "0.10.3"
```

File: nativescript-angular/package.json (hunk 2 above)
Comment (vakrilov) [inline nativescript-angular/package.json]: No point to have them as a dev-deps. I'll move them

File: nativescript-angular/package.json (hunk 3 above)
Comment (vakrilov) [inline nativescript-angular/package.json]: Yup

**Pull request merged**